google/cloud/internal/oauth2_external_account_credentials.cc

```diff
@@ -148,6 +157,15 @@ StatusOr<AccessToken> ExternalAccountCredentials::GetToken(
       {"subject_token_type", info_.subject_token_type},
       {"subject_token", subject_token->token},
   };
+
+  // Workforce Identity is handled at org level, it requires userProject in
```

Comment:
nit: "Workforce Identity is handled at the org level and requires the `userProject` header. Workload Identity is handled at the project level and doesn't require the header."